packages/jaeger-ui/src/model/transform-trace-data.test.js

```diff
@@ -76,7 +76,13 @@ describe('transformTraceData()', () => {
       ],
       startTime,
       duration,
-      tags: [],
+      tags: [
+        {
+          key: "bigintkey",
+          type: "int64",
+          value: 780177723283783680
```

Comment:
a) this isn't a test by itself since it's not checking anything
b) defining data in code is not equivalent to parsing the data from a JSON